PHC-Android/PHC/src/main/java/phc/android/Checkout/CheckoutScannerFragment.java

```diff
@@ -48,11 +50,13 @@ protected void confirmScan(CharSequence scanResult, boolean manualInput) {
         args.putBoolean("manual_input", manualInput);
 //        args.putStringArrayList("services_not_received", mServicesNotReceived);
 
-        Fragment confFrag = new CheckoutFormFragment();
-        confFrag.setArguments(args);
-//        displayNextFragment(confFrag, CheckoutFormFragment.TAG);
+        Fragment scannerConfFrag = new CheckoutScannerConfirmationFragment();
+        scannerConfFrag.setArguments(args);
+        displayNextFragment(scannerConfFrag, CheckoutScannerConfirmationFragment.TAG);
     }
 
+
+
     private class LoginResponseListener implements Response.Listener<JSONObject> {
```

Comment:
What are these responselisteners for? I don't believe this fragment makes any network requests?